test/test/sklearndf/test_sklearn_coverage.py

```diff
@@ -57,11 +59,20 @@
     *GENERAL_COVERAGE_EXCLUSIONS,
     # class "Imputer" was deprecated in 0.20 and removed in 0.22
     "Imputer",
+    "AgglomerationTransform",  # TODO should be?
 }
 
 
 PIPELINE_COVERAGE_EXCLUSIONS = GENERAL_COVERAGE_EXCLUSIONS
 
+
+CLUSTERER_COVERAGE_EXCLUSIONS = {
+    *GENERAL_COVERAGE_EXCLUSIONS,
+    # class "FeatureAgglomeration" is already included via Transformers
+    "FeatureAgglomeration",
```

Comment:
Shouldn't we also test `FeatureAgglomeration` as part of the clustering tests? We want to confirm that FeatureAgglomeration also implements `ClustererDF`.

Reply:
To make tests for clusterers pass then I need to also wrap `FeatureAgglomeration` in clusterer module. But then we will have redundancy as `FeatureAgglomeration` will be present in clusterers and transformers:
```
from sklearndf.transformation import FeatureAgglomerationDF
from sklearndf.clustering import FeatureAgglomerationDF
```

I think we can either leave `FeatureAgglomeration` in transformers as is and keep that exclusion or move `FeatureAgglomeration` to clusterers (but then I need to get `ColumnPreservingTransformerWrapperDF` in clusterers package which is implemented in transformers package that this algorithm uses).

Should I leave it as is or move `FeatureAgglomeration` to clusterers?

Reply:
Agree if this is just about testing coverage and not testing functionality such as `fit_predict`.
`FeatureAgglomeration` lives in `sklearn.cluster` so perhaps move it to clusters also in our test code?

Reply:
I took another look at it and I think this one is a little bit more complex. So in `sklearn` `FeatureAgglomeration` that is placed in `clustering` implements both `ClusterMixin` and `TransformerMixin` and it's both clusterer and transformer. If I move it to `clustering` package and wrap it in `ClustererDF` it will lost wrapping for `transform` and `inverse_transform` methods. 

My idea is to leave it where it is for this PR and prepare another one where:
1.  `FeatureAgglomeration` will be moved to `clustering` package
2. It will be wrapped in such a way that it will be both `TransformerDF` and `ClustererDF` (I need to see if that's going to be even feasible without too much hacking)

What do you think?